picard/cluster.py

```diff
@@ -227,6 +227,9 @@ def can_view_info(self):
     def can_submit(self):
         return not self.special and self.files
 
+    def can_extract(self):
```

Comment:
Like other of those feature properties I think `can_extract should be defined for the `Item` class (in `ui/item.py`). This avoids that you need to declare it in all those item subclasses.

Reply:
Done.